Update dotnet-core.md to remove misleading statement in installation … (#23024)
DataDog/documentation: content/en/tracing/trace_collection/automatic_instrumentation/dd_libraries/dotnet-core.md

</div>

<div class="alert alert-warning">

>   <strong>Note:</strong> Datadog's automatic instrumentation relies on the .NET CLR Profiling API. This API allows only one subscriber, with exception of our Datadog APM and the Continous Profiler. To ensure maximum visibility, run only one APM solution in your application environment. 

Reviewer: ```suggestion
  <strong>Note:</strong> Datadog's automatic instrumentation relies on the .NET CLR Profiling API. This API allows only one subscriber, with the exception of APM and the Continuous Profiler. To ensure maximum visibility, run only one APM solution in your application environment. 
```

Reviewer 2: What do you think of this suggestion? I think it might be confusing to say that there is are exceptions. Additionally, whatever we end up with in this file, can you please copy the contents over to the dotnet-framework page as well?
```suggestion
  <strong>Note:</strong> Datadog's automatic instrumentation relies on the .NET CLR Profiling API. This API allows only one subscriber (for example, Datadog APM). To ensure maximum visibility, run only one APM solution in your application environment. 
```

Author: We can go with that to keep it simple. I agree.
